src/content/drafts/eslint-community-org.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Announcing the ESLint Community GitHub organization
+teaser: >-
+  The ESLint Community GitHub organization is a place for important ecosystem
+  projects to get the attention and support they deserve.
+tags:
+  - eslint-community
+  - Updates
+authors:
+- MichaelDeBoey
+categories:
+  - Announcements
+---
+
+In our
+[2022 year in review](https://eslint.org/blog/2023/01/eslint-2022-year-review),
+we briefly mentioned a couple of project kickoffs, and we're excited to now
+announce the official
+[ESLint Community GitHub organization](https://github.com/eslint-community).
+
+The strength of ESLint is in its ecosystem, and we continue to investigate
+different ways of further supporting that community. We began by
+[donating to community projects](https://eslint.org/blog/2022/02/paying-contributors-sponsoring-projects/#supporting-the-community),
+to ensure that high-value projects were receiving the funds they needed to
+continue with ongoing maintenance and development. Our next step is to create
+the `@eslint-community` GitHub organization as a home for high-value projects in
+the ESLint ecosystem.
+
+As you can read in the
+["`@eslint-community` GitHub organization" RFC](https://github.com/eslint/rfcs/tree/main/designs/2022-community-eslint-org),
+the goal of this new organization is to have a place where community members can
+help ensure widely depended upon ESLint-related packages live and never fall out
+of maintenance.  
+The Community Core team (which currently consists of
+[`@aladdin-add`](https://github.com/aladdin-add),
+[`@ota-meshi`](https://github.com/ota-meshi),
+[`@voxpelli`](https://github.com/voxpelli) & myself
+[`@MichaelDeBoey`](https://github.com/MichaelDeBoey)) & all package maintainers
+will make sure these packages stay up-to-date with newer ESLint releases, and
+they don't hold the wider community back by depending on a single person's
+GitHub or `npm` account.
+
+You might have noticed that some of your most beloved ESLint-related packages
+have already been moved over to or been forked by the `eslint-community`
+organization. A full list of the current projects include:
+
+* [`eslint-formatter-codeframe`](https://github.com/eslint-community/eslint-formatter-codeframe)
+  &
+  [`eslint-formatter-table`](https://github.com/eslint-community/eslint-formatter-table)  
+  We took these over from [@fregante](https://github.com/fregante)
+* [`eslint-plugin-eslint-plugin`](https://github.com/eslint-community/eslint-plugin-eslint-plugin)  
+  [@not-an-aardvark](https://github.com/not-an-aardvark) transferred this repo
+  to the new org, but is still maintaining it
```

Comment:
just want to point out: @bmish has been the active maintainer for a while. 😄

Reply:
Thanks @aladdin-add! Appreciate your support with it. Yes, I've been the primary maintainer on eslint-plugin-eslint-plugin for 2 years.